src/imitation/scripts/common/train.py

```diff
@@ -32,6 +33,13 @@ def fast():
     locals()  # quieten flake8
 
 
+@train_ingredient.named_config
+def sac():
+    policy_cls = SACPolicy
+
+    locals()  # quieten flake8
```

Comment:
for a one-liner, just adding `# noqa: F841` is cleaner

Reply:
Sure!